wallet/lib/pages/presenting_onboard_page/screens/accept_policy_screen.dart

```diff
@@ -0,0 +1,249 @@
+                              height: 15.0.h,
+                            ),
+                            MyCheckBox(
+                              LocaleKeys.acknowledge_privacy_policy.tr(),
+                              isSelected: viewModel.isCheckPrivacyPolicy,
+                              onChange: (value) {
+                                viewModel.toggleCheckPrivacyPolicy(value!);
+                              },
+                              onLinkTap: () => launchUrlString(kPrivacyPolicyLink),
+                            ),
+                            SizedBox(
+                              height: 25.0.h,
+                            ),
+                            PylonsGetStartedButton(
+                              key: const Key(kAcceptBottomSheetBtnKey),
+                              enabled: viewModel.isCheckTermServices && viewModel.isCheckPrivacyPolicy,
+                              onTap: () async {
+                                viewModel.setUserAcceptPolicies();
+                                await navigatorKey.currentState!.pushNamed(RouteUtil.ROUTE_PURCHASE_VIEW, arguments: widget.nft);
+                              },
+                              text: LocaleKeys.get_started.tr(),
+                              loader: ValueNotifier(false),
+                              fontWeight: FontWeight.normal,
+                              btnHeight: 40,
+                              btnWidth: 260,
+                              btnUnselectBGColor: AppColors.kDarkDividerColor,
+                              fontSize: 14,
+                              textColor: !(viewModel.isCheckTermServices && viewModel.isCheckPrivacyPolicy) ? AppColors.kUserInputTextColor : AppColors.kWhite,
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  );
+                },
+              );
+            },
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget getTypeWidget(NFT nft) {
+    switch (nft.assetType) {
+      case AssetType.Image:
+        return imageWidget(nft.url);
+      case AssetType.Audio:
+      case AssetType.Video:
+      case AssetType.Pdf:
+        return imageWidget(nft.thumbnailUrl);
+      case AssetType.ThreeD:
+        return Container(
+          color: Colors.grey.shade200,
+          height: double.infinity,
+          child: Nft3dWidget(
+            url: nft.url,
+            cameraControls: false,
+            backgroundColor: AppColors.kBlack,
+            showLoader: true,
+          ),
+        );
+
+      default:
+        return imageWidget(nft.url);
+    }
+  }
+
+  CachedNetworkImage imageWidget(String url) {
+    return CachedNetworkImage(
+      placeholder: (context, url) => Shimmer(color: PylonsAppTheme.cardBackground, child: const SizedBox.expand()),
+      imageUrl: url,
+      fit: BoxFit.fill,
+    );
+  }
+}
+
+class MyCheckBox extends StatelessWidget {
```

Comment:
I think there is already a checkbox in our create key flow cant we reuse it?

Reply:
That checkbox we have in out create key flow, there design is change and i redesigned checkbox for accept policy screen according to design.